user/views.py

```diff
@@ -81,3 +87,74 @@ def post(self, request):
 
         except KeyError:
             return JsonResponse({"message" : "KEY_ERROR"}, status=400)
+
+class UserLikeView(View):
+    @login_decorator
+    def get(self, request):
+        if not User.objects.filter(id=request.user.id):
+            return JsonResponse({"message" : "INVALID_USER"})
+        
+        user = User.objects.get(id=request.user.id)
+
```

Comment:
```suggestion
        
        user = request.user

```
로그인 데코레이터에서 request에 이미 user 인스턴스를 담아서 보냈기 때문에 다시 User 클래스에 접근하시는 방법은 쿼리가 하나 낭비되는 과정을 만듭니다 :) 그래서 바로 request에서 user를 꺼내오시기만 하면 됩니다!

Reply:
넵!! 바로 꺼내오겠습니다~🧤